multimedia/ffmpeg6/Portfile

```diff
@@ -323,9 +323,11 @@ platform darwin {
 
 # libavcodec/pcm-bluray.c:172:45: error: passing argument 2 of 'bytestream2_get_buffer' from incompatible pointer type [-Wincompatible-pointer-types]
 # libavcodec/pcm-dvd.c:155:37: error: passing argument 2 of 'bytestream2_get_buffer' from incompatible pointer type [-Wincompatible-pointer-types]
+# vulkan_av1.c:216:43: error: initialization of 'long long unsigned int' from 'void *' makes integer from pointer without a cast [-Wint-conversion]
 if {[string match *gcc* ${configure.compiler}]} {
     configure.cflags-append \
-                    -Wno-error=incompatible-pointer-types
+                    -Wno-error=incompatible-pointer-types \
+                    -Wno-error=int-conversion
```

Comment:
Ideally we should fix code issues, rather than ignore compilation errors. (Though to be fair, I haven't always been the best role model in that regard.)

But for now, I suppose this is okay. Sigh...